bundle/DependencyInjection/NetgenEzPlatformSiteApiExtension.php

```diff
@@ -78,22 +75,4 @@ public function prepend(ContainerBuilder $container): void
 
         $container->prependExtensionConfig('ezpublish', $config);
     }
-
-    /**
-     * Default values must be set conditionally because eZ Kernel bundles are usually activated
-     * before this extension is ran. In that case the values that were already processed and merged
-     * to the container would be overwritten with the default value.
-     *
-     * @param \Symfony\Component\DependencyInjection\ContainerBuilder $container
-     */
-    private function setDefaultValuesIfNeeded(ContainerBuilder $container): void
```

Comment:
Will this change make sure that we do not get back the issue described in the comment here?

Reply:
It should, this is what eZ's bundles use. But TBH I didn't test it, will try to reproduce it.

Reply:
Reproduced and validated, this fixes the problem.